FAQ.md

```diff
@@ -39,10 +39,11 @@ Intel processors. AMD and ARM processors will be supported in the near future.
 
 ### Can Firecracker be used with Kubernetes, Docker, or Kata containers today?
 
-Not yet. We are making Firecracker open source because it provides a
-meaningfully different approach to security for running containers. We hope that
-others in the communities that build open source container technology find it
-useful. We are working to make Firecracker integrate naturally with the
+Not yet with Kubernetes and Docker, but in release
```

Comment:
Since this is within the hypervisor repository, I think it'd be more accurate to say that it can be used with Docker and Kubernetes (limited use cases) today via Kata, rather than say that it is not yet possible (not accurate).  To give a nod to the firecracker-containerd work, we can say that other integration is still a WIP.  

Thoughts?  @samuelkarp @nmeyerhans

Reply:
Yeah, I might say something indicating that we intend to facilitate the use of Firecracker with Docker, k8s, and other container orchestrators, but that the work is outside the scope of Firecracker itself. You can then include references to existing integration efforts for further information.

Reply:
FAQ updated.